### PR TITLE
Update project status event with troubleshoot info

### PR DESCRIPTION
**Pull request by lentheivendra**
For https://github.com/eclipse/codewind/issues/1812

Updated project status event to include troubleshooting info as discussed, and tested changes by creating Appsody node.js project which recreates scenario of app stuck in starting state.

Signed-off-by: Len Theivendra <theivend@ca.ibm.com>

### CODE REVIEW

**Comment by rajivnathan** [comment]
@jopit @elsony Do you think we should add another optional property to `detailedAppStatus` like `statusId` or something so that IDEs can differentiate between status events in case we want to do something like fine-grained filtering of these events if needed?

**Comment by jopit** [comment]
@rajivnathan would the status id identify the type of detailedStatus, like "ping timed out" or would it identify a set of related detailedStatus messages -- i.e. an unbroken sequence of "ping timed out" errors would get one id, then if ping worked for a while and then broke again, there'd be a new status id for the next unbroken sequence of "ping timed out" messages?

**Comment by rajivnathan** [comment]
@jopit I was thinking the former, so just a general type of detailedStatus eg. "ping timed out". But after talking it through with Elson I think we can put that in if/when we decide we need something like that.

File: src/pfe/file-watcher/server/src/controllers/projectStatusController.ts [hunk 1]
```diff
@@ -530,10 +532,13 @@ function pingInTransitApplications(): void {
                                         data.detailedAppStatus = detailedAppStatus;
                                         pingCountMap.delete(projectID);
                                     } else if (newState === AppState.starting && pingCount == pingCountLimit) {
+                                        const troubleShootLinkLabel = await locale.getTranslation("projectStatusController.linkLabel");
                                         detailedAppStatus = {
                                             severity: "WARN",
                                             message: newMsg,
-                                            notify: true
+                                            notify: true,
+                                            linkLabel: troubleShootLinkLabel,
```
Comment (rajivnathan) [inline src/pfe/file-watcher/server/src/controllers/projectStatusController.ts]: Do we need updates to the unit/functional tests to validate that these events have the `linkLabel` and `link` properties when `notify` is `true`?

Comment (rajivnathan) [reply src/pfe/file-watcher/server/src/controllers/projectStatusController.ts]: We can handle this in the subsequent PR

**Pull request merged**